RELEASE_BLOG_POST_TEMPLATE.md.bak

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Did you bring this template back intentionally? Why are there two?

Reply:
> Did you bring this template back intentionally? Why are there two?

No, I've deleted it now.